llvm/test/CodeGen/X86/xray-conditional-tail-call.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: llc -mtriple=x86_64 < %s | FileCheck %s
```

Comment:
Reuse xray-tail-call-sled.ll 

https://maskray.me/blog/2021-08-08-toolchain-testing#i-dont-know-an-existing-test-can-be-enhanced